_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -1057,6 +1070,16 @@ published_cluster_states | Object | Statistics for the published cluster states
 published_cluster_states.full_states | Integer | The number of published cluster states.
 published_cluster_states.incompatible_diffs | Integer | The number of incompatible differences between published cluster states.
 published_cluster_states.compatible_diffs | Integer | The number of compatible differences between published cluster states.
+cluster_state_stats | Object | Cluster state updates statistics published by active leader.
+cluster_state_stats.overall | Object | Cluster state update statistics including local and all remote state uploads done by the node.
```

Comment:
can we just say -> Overall cluster state updates statistics

Reply:
As long as it's easy to understand that overall time includes both local state update as well as remote upload. Because currently `cluster_state_stats.overall.total_time_in_millis` doesn't signify anything specifically. Is it okay ?

```
cluster_state_stats.overall.total_time_in_millis | Integer | The total amount of time for all cluster state updates, in milliseconds.
```